ubuntu-github-actions-android/Dockerfile-30.0.3

```diff
@@ -0,0 +1,71 @@
+# Version: 20231114
+# Image name: quay.io/opencv-ci/opencv-androidsdk-30
+# For 4.x branch
+
+FROM ubuntu:20.04
+
+ARG GRADLE_VERSION=7.5.1
+ARG ANDROID_BUILD_TOOLS_VERSION=30.0.3
+ARG ANDROID_PLATFORM_VERSION=android-26
+ARG ANDROID_TOOLS_ARCHIVE=commandlinetools-linux-6858069_latest.zip
+ARG NDK_VERSION=18.1.5063045
+ARG CMAKE_VERSION=3.22.1
+
+USER root
+
+ENV DEBIAN_FRONTEND noninteractive
+
+RUN \
+  apt-get update && \
+  apt-get install -y --no-install-recommends \
+    software-properties-common \
+    build-essential ccache ninja-build \
+    python3 python3-pip curl zip unzip file \
+    openjdk-8-jdk openjdk-17-jre libncurses5 tree \
+    && \
+  add-apt-repository ppa:git-core/ppa && \
+  apt-get install -y --no-install-recommends git && \
+  rm -rf /var/lib/apt/lists/*
+
+RUN mkdir -p /opt/gradle && \
```

Comment:
Is it possible to install system gradle package? I thought that required version will be automatically downloaded by project if necessary.